Makefile

```diff
@@ -1,7 +1,8 @@
-.PHONY: syntax-check
-syntax-check:
-	@find streams updates -iname '*.json' | sort | xargs -n 1 python3 -c 'import json, sys; json.load(open(sys.argv[1]))'
+.PHONY: syntax-check pre-commit-check print-rollouts
 
-.PHONY: print-rollouts
+syntax-check: pre-commit-check
+	@find streams updates -iname '*.json' | sort | xargs -n 1 python3 -c 'import json, sys; json.load(open(sys.argv[1]))'
+pre-commit-check:
+	pre-commit run --all-files
```

Comment:
```suggestion
syntax-check: pre-commit-check
	@find streams updates -iname '*.json' | sort | xargs -n 1 python3 -c 'import json, sys; json.load(open(sys.argv[1]))'
	@find release-notes -iname '*.yaml' | sort | xargs -n1 python3 ci/check-release-notes.py
```

Reply:
thanks for this input. I have made these changes.